importer/MtDe.py

```diff
@@ -0,0 +1,96 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class MtDe(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Use the 'gemeinde' field as
+        municipality.
+        """
+        municip_raw = self.gemeinde
+        municip_q = utils.q_from_wikipedia("de", municip_raw)
+        if municip_q:
+            self.add_statement("located_adm", municip_q)
+        else:
+            self.add_to_report("gemeinde", self.gemeinde, "located_adm")
+
+    def update_labels(self):
+        labels = {
+            "de": self.name_de,
+            "en": self.name_en,
+            "mt": self.name_mt
+        }
+        for language in labels:
+            self.add_label(language, labels[language])
+
+    def update_descriptions(self):
+        """
+        Set the descriptions in English.
+
+        Try to add the place name. Since the municipality
+        names are titles of articles, sometimes they
+        have the form of "$municipality (Malta)", in which
+        case remove the bracketed part.
+        """
+        placename = "Malta"
+        description = "cultural property in {}"
+        if self.has_non_empty_attribute("gemeinde"):
+            municip_base = utils.get_rid_of_brackets(self.gemeinde)
+            placename = "{}, {}".format(municip_base, "Malta")
+        self.add_description("en", description.format(placename))
+
+    def set_monuments_all_id(self):
+        self.monuments_all_id = self.inventarnummer
+
+    def set_heritage_id(self):
+        self.add_statement("malta_monument_id", self.inventarnummer)
+
+    def set_street_address(self):
+        """
+        Set the street address.
+
+        These all have nice looking street addresses, which, however,
+        do not include the municipality name.
+        So add it to the address after stripping potential
+        bracketed part.
+        """
+        if utils.contains_digit(self.adresse):
+            city_clean = utils.get_rid_of_brackets(self.gemeinde)
+            address = "{}, {}".format(self.adresse, city_clean)
+            self.add_statement("located_street", address)
+
+    def exists_with_monument_article(self, language):
```

Comment:
A question which also affects other coutnries. What hapens with a monuments_article of the form "Victoria (Malta)#Zitadelle"? (that is inventarnummer=02 for malta)

The correct behaviour should be that it is interpreted as there being no value at all (since we cannot make a connection between an article section and a Wikidata item).

Reply:
Added a check to the function in Monument so that it ignores titles with #.